x/events/events.go

```diff
@@ -0,0 +1,112 @@
+// Copyright © 2023 Ory Corp
+// SPDX-License-Identifier: Apache-2.0
+
+package events
+
+import (
+	"context"
+	"reflect"
+
+	"github.com/gofrs/uuid"
+	otelattr "go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
+
+	f "github.com/ory/kratos/selfservice/flow"
+
+	"github.com/ory/x/otelx/semconv"
+)
+
+const (
+	SessionIssued      semconv.Event = "SessionIssued"
+	SessionRevoked     semconv.Event = "SessionRevoked"
+	RegistrationFailed semconv.Event = "RegistrationFailed"
+	IdentityCreated    semconv.Event = "IdentityCreated"
+	LoginFailed        semconv.Event = "LoginFailed"
```

Comment:
Any reason to make these `semconv.Event`s instead of just strings?

Reply:
Helps with type safety [here](https://github.com/ory-corp/cloud/blob/651ca4b708c65e38f71be9aded34a7aeb6ec6691/backoffice/backoffice/projectstats/projectstats.go#L169)